models/lobby.go

```diff
@@ -193,21 +207,25 @@ func (lobby *Lobby) AddPlayer(player *Player, slot int) *helpers.TPError {
 
 	db.DB.Create(newSlotObj)
 
-	lobby.updateServerAllowedPlayers()
-
+	if !config.Constants.ServerMockUp {
+		Pauling.Call("Pauling.AllowPlayer", &Args{Id: lobby.ID, SteamId: player.SteamId}, &Args{})
+	}
 	return nil
 }
 
 func (lobby *Lobby) RemovePlayer(player *Player) *helpers.TPError {
 	err := db.DB.Where("player_id = ? AND lobby_id = ?", player.ID, lobby.ID).Delete(&LobbySlot{}).Error
-	lobby.updateServerAllowedPlayers()
 	if err != nil {
 		return helpers.NewTPError(err.Error(), -1)
 	}
+
 	return nil
 }
 
 func (lobby *Lobby) BanPlayer(player *Player) {
+	if !config.Constants.ServerMockUp {
+		Pauling.Call("Pauling.DisallowPlayer", &Args{Id: lobby.ID, SteamId: player.SteamId}, &Args{})
```

Comment:
I'd prefer if calls to Pauling went through an additional layer to give type safety and autocompletion to Pauling methods. Something like `paulingint.DisallowPlayer(lobby.ID, player.SteamId)` that would then do `Pauling.Call("Pauling.DisallowPlayer", &Args{Id: lobby.ID, SteamId: player.SteamId}, &Args{})`